dotcom-rendering/src/components/InteractivesDisableArticleSwipe.importable.tsx

```diff
@@ -3,12 +3,31 @@ import { useEffect } from 'react';
 import { getInteractionClient } from '../lib/bridgetApi';
 
 export const InteractivesDisableArticleSwipe = () => {
-	useEffect(() => {
-		void getInteractionClient()
+	const onTouchStart = () => {
+		getInteractionClient()
 			.disableArticleSwipe(true)
 			.catch((error) => {
-				log('dotcom', 'disableArticleSwipe failed:', error);
+				log('dotcom', 'disableArticleSwipe(true) failed', error);
+			});
+	};
+
+	const onTouchEnd = () => {
+		getInteractionClient()
+			.disableArticleSwipe(false)
+			.catch((error) => {
+				log('dotcom', 'disableArticleSwipe(false) failed', error);
 			});
+	};
+	useEffect(() => {
+		document.addEventListener('touchstart', onTouchStart, {
+			passive: true,
+		});
+		document.addEventListener('touchend', onTouchEnd, { passive: true });
+
+		return () => {
+			document.removeEventListener('touchstart', onTouchStart);
+			document.removeEventListener('touchend', onTouchEnd);
+		};
 	}, []);
 	return null;
```

Comment:
Also wondering if you tested the number of events that are being fired. I think this might cause too many events being fired and might have performance side effect. I'm not sure though, it needs testing

Reply:
I haven't, it's tricky to get things in the emulators pre-beta, though it shouldn't fire any more than carousels or key events on live blogs should it? A blanket on/off switch would be ideal but my understanding is that's not how the Android implementation works

Reply:
I will setup my android simulator now and will test this. Will let you know 👍